packages/peregrine/lib/store/actions/user/asyncActions.js

```diff
@@ -39,9 +39,7 @@ export const getUserDetails = ({ fetchUserDetails }) =>
             dispatch(actions.getDetails.request());
 
             try {
-                const { data } = await fetchUserDetails({
-                    fetchPolicy: 'no-cache'
-                });
+                const { data } = await fetchUserDetails();
```

Comment:
This change was me; since we're already fetching the data and storing in redux, I just removed this fetch policy so it also caches the result in Apollo. We plan to cleanup all this legacy code in scope of [PWA-321](https://jira.corp.magento.com/browse/PWA-321), so this just handled this one case.